File: SOSAuto_FRONTEND/src/app/services/models/participation-dto.ts
/* tslint:disable */
/* eslint-disable */
import { DemandeDepannage } from '../models/demande-depannage';
import { MecanicienDto } from '../models/mecanicien-dto';
export interface ParticipationDto {
  demande?: DemandeDepannage;
  id?: number;
  mecanicien?: MecanicienDto;
  status?: 'EN_ATTENTE' | 'ACCEPTE' | 'REFUSE' | 'ANNULER';
}

File: SOSAuto_BACKEND/src/main/java/com/sosauto_backend/model/dto/ParticipationDTO.java
package com.sosauto_backend.model.dto;

import com.sosauto_backend.model.entity.DemandeDepannage;
import com.sosauto_backend.model.enums.StatutParticipation;
import lombok.Data;

@Data
public class ParticipationDTO {

        private Long id;
        private DemandeDepannage demande;
        private MecanicienDTO mecanicien;
        private StatutParticipation status;


}
